### ISSUE TITLE
Restart failed

### ISSUE THREAD

**Issue by MarkYSA** [leoarnold/puppet-cups#63]
Hello.

I've faced with some issue related to the restart process of cupsd on Ubuntu 18.04.1:

I always receive this message while puppet perform cupsd process restart:

Failed to apply catalog:
```
IPP query 'ipptool -c ipp://localhost/ /dev/stdin' failed.
EXITCODE: 1
STDIN:
{
  OPERATION CUPS-Get-Classes
  GROUP operation
  ATTR charset attributes-charset utf-8
  ATTR language attributes-natural-language en
  DISPLAY printer-name DISPLAY member-names
}
STDOUT:
STDERR: ipptool: Unable to connect to localhost on port 631 - Operation now in progress
```

But in the real the daemon is restarted and second puppet run always successful and returns no errors.

Looks like we need to implement something like short timeout here.

Regards, Mark

**Comment by leoarnold** [comment]
@MarkYSA The module does not yet clain to support Ubuntu 18.04 or above:

![Puppet Forge](https://user-images.githubusercontent.com/4201235/47142205-b2cbf700-d2c2-11e8-94fe-97783e621c2c.png)

Nevertheless I already ran the acceptance tests on a VM with Ubuntu 18.04 and they all passed, so I'm not sure yet if I can reproduce this.

We have seen such behavior before when Ubuntu 16.04 came out and it was due to the way `systemd` runs daemons. Interestingly enough, this problem disappeared somehow - probably was fixed on Ubuntu's side.

What is new about your error is the additional message `Operation now in progress`. I'll have to research what that wants to tell us.
